EIPS/eip-5792.md

```diff
@@ -358,17 +363,34 @@ type AtomicCapability = {
 {
   "0x2105": {
     "atomic": {
-      "supported": true
+      "status": "supported"
     }
   },
   "0x14A34": {
     "atomic": {
-      "supported": false
+      "supported": "unsupported"
     }
   }
 }
 ```
 
+### Error Codes
+
+The following error codes are defined for the methods specified in this EIP. These errors follow the JSON-RPC 2.0 specification for error objects, which require a `code` and `message` field.
+
+| Code | Message | Description | Related RPCs |
+|------|---------|-------------|--------------|
+| 4001 | User Rejected Request | The user rejected submitting the batch of calls (from [EIP-1193](./eip-1193.md)) | `wallet_sendCalls` |
+| 4100 | Unauthorized | The specified address is not connected or not in the wallet (from [EIP-1193](./eip-1193.md)) | `wallet_sendCalls`, `wallet_getCapabilities` |
```

Comment:
[nit]: should the description match the EIP-1193 description for these errors?

Reply:
would the `(from [EIP-1193](./eip-1193.md))` be sent as part of the message? i assume not but, might be clearer to move that to another column